pop-api/src/lib.rs

```diff
@@ -27,14 +28,18 @@ pub type Result<T> = core::result::Result<T, PopApiError>;
 #[derive(Debug, Copy, Clone, PartialEq, Eq, scale::Encode, scale::Decode)]
 #[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
 pub enum PopApiError {
-    PlaceholderError,
+    RuntimeError,
+    Nfts(nfts::Error),
 }
 
 impl ink::env::chain_extension::FromStatusCode for PopApiError {
     fn from_status_code(status_code: u32) -> core::result::Result<(), Self> {
         match status_code {
             0 => Ok(()),
-            1 => Err(Self::PlaceholderError),
+            1 => Err(Self::RuntimeError),
+            50_000..=50_999 => {
```

Comment:
We should probably introduce a const NFT_INDEX = 50.

Reply:
Gave it a go, it doesnt like (`nfts::PALLET_INDEX * 1_000`) in the match arm so had to revert.